### ISSUE TITLE
fgDisabled on light theme fails contrast

### ISSUE THREAD

**Issue by six7** [tokens-studio/figma-plugin#2636]
We need to fix fgDisabled for light theme in @tokens-studio/tokens

This is mainly a color contrast related issue, and needs to be solved in github.com/tokens-studio/ds.

There we have a packages/tokens folder. Inside of that there's our various tokens stored. The light theme's token decision seems to refer to something that fails color contrast. When creating the PR make sure there's a changeset (with `yarn changeset`) so that when the PR is merged it will automatically create a versioned release on npm

![Image](https://github.com/tokens-studio/figma-plugin/assets/4548309/724bc42a-b626-4983-bf09-e79f393e6077)
